javascript/net/grpc/web/generator/grpc_generator.cc

```diff
@@ -1522,9 +1523,12 @@ class GrpcCodeGenerator : public CodeGenerator {
 
   uint64_t GetSupportedFeatures() const override {
     // Code generators must explicitly support proto3 optional.
-    return CodeGenerator::FEATURE_PROTO3_OPTIONAL;
+    return CodeGenerator::FEATURE_PROTO3_OPTIONAL | CodeGenerator::FEATURE_SUPPORTS_EDITIONS;
   }
 
+  Edition GetMinimumEdition() const override { return Edition::EDITION_PROTO2; }
+  Edition GetMaximumEdition() const override { return Edition::EDITION_2023; }
```

Comment:
Is this matching what protobuf-javascript supports? 😃

If so, do you mind adding a note linking to it so we can keep this up to date?

Reply:
Great! Thank you! 😃